onoff.d.ts

```diff
@@ -0,0 +1,44 @@
+
+export class Gpio {
+    static HIGH: number;
+    static LOW: number;
+    static accessible: boolean;
+
+    constructor(gpio: number, direction: Directions, edge?: Edges, options?: Options);
+
+    read(callback: ValueCallback): void;
+    read(): Promise<BinaryValue>;
+
+    readSync(): BinaryValue;
+
+    write(value: BinaryValue, callback: (err: Error | undefined) => void): void;
```

Comment:
I'm probably about display my lack of knowledge about TypeScript now, but should this:
```ts
err: Error | undefined
```
not be this:
```ts
err: Error | null | undefined
```

Reply:
If null is a possible value then yes it should be explicitly mentioned.
Reading back in our conversation, yes I seem to have overlooked this case.

Reply:
The `err` in this case comes from [the fs.write call](https://github.com/fivdi/onoff/blob/v4.0.0/onoff.js#L195) and as mentioned [file system documentation](https://nodejs.org/dist/latest-v11.x/docs/api/fs.html#fs_file_system) it can be `null` or `undefined`.